### ISSUE TITLE
Ctrl+7 (opening Code Block) should set focus to the Code Block

### ISSUE THREAD

**Issue by Biepa** [BookStackApp/BookStack#1972]
**Describe the feature you'd like**
When I edit a page and want to insert a new code block I press ctrl+7.
Then i start typing, but the cursor is still typing on the main page in the background, so everytime I have to take my mouse to focus the code block. So an automatic focus, would be nice.

Edit: Maybe also an option to save the code block with a shortcut would also help (ctrl + Enter, maybe?).

**Describe the benefits this feature would bring to BookStack users**
Faster creating and editing with code blocks -> better experience

**Comment by Biepa** [comment]
To make the workflow perfect.
After closing the code-block editor it would be cool, if the cursor would be behind the inserted code-block.

**Comment by ssddanbrown** [comment]
Thank you for the suggestion @Biepa.

I have just implemented some updates, these will be release in v0.29. I've ensured the whole code-insert flow can be done via keyboard, So: 
- Code editor in popup will gain focus on enter.
- Code in editor can be saved using `ctrl+enter`.
- WYSIWYG page editor will get back focus on save.

The last step, page editor refocus, can be a bit quirky. It mostly lands the cursor at the start of the code block but in some cases, such as when immediately after another code block, it can jump to the start.
